lib/assets/aws/index.js

```diff
@@ -15,7 +17,7 @@ module.exports = (function(){
         LOGGING_CATEGORY = 'AWS-ASSET-MANAGER';
 
     internal.buildKey = function(nodeid, filename, prefix) {
-        var result = (internal.config.assetsDir ? internal.config.assetsDir + '/' : '') + nodeid + (filename ? '/' + filename : '');
```

Comment:
`grasshopper` is undefined, and `replace(' '` will only work on the first one. updated to `/\s/g`

Reply:
was able to test that filenameSpaceReplacement works, but for whatever reason the demo app gh admin is sending back `undefined` as the node id whenever I try to delete something... merging since it looks like it should work and then I'll keep working on why the demo repo seems broken.